sdcm/provision/common/utils.py

```diff
@@ -192,3 +192,15 @@ def install_syslogng_exporter():
     systemctl enable syslog_ng_exporter.service
     systemctl start syslog_ng_exporter.service
 """)
+
+
+def disable_daily_apt_triggers():
+    return dedent("""\
+    if apt-get --help >/dev/null 2>&1 ; then
```

Comment:
We don't have the equivalent for CentOS/rhel/rocky ?

Other disablement of things that can move into this part ?

Reply:
No, rhel based ones don't have it - for them we disable firewall only. I think moving it won't bring much improvement.

Reply:
o.k. lets leave it as is